### PR TITLE
Add custom pyfunc blog

### PR DESCRIPTION
**Pull request by BenWilson2**
Resubmit with fixes (convert all to log_model()) from renamed fork

### CODE REVIEW

**Comment by harupy** [comment]
@BenWilson2 Can you rebase on main?

File: website/blog/2024-01-23-custom-pyfunc/index.md [hunk 1]
```diff
@@ -0,0 +1,599 @@
+---
+title: Custom MLflow Models with mlflow.pyfunc
+description: A guide for creating custom MLflow models
+slug: custom-pyfunc
+date: 2024-01-23
+categories: blog
```
Comment (harupy) [inline website/blog/2024-01-23-custom-pyfunc/index.md:6]:
```suggestion
```

File: website/blog/2024-01-23-custom-pyfunc/index.md [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+with mlflow.start_run():
+    model_info = mlflow.pyfunc.log_model(
+        artifact_path="my_custom_model",
+        python_model=<model>,
+        # Specify where artifacts can be found
+        artifacts={
+            "artifact": <artifact_1_path>,
+            "another_artifact": <another_artifact_path>
+        },
+        [...]
```
Comment (harupy) [inline website/blog/2024-01-23-custom-pyfunc/index.md:347]: Is this line valid code?

**Review by harupy — APPROVED** [review]
LGTM

File: website/blog/2024-01-23-custom-pyfunc/index.md (hunk 1 above)
Comment (BenWilson2) [reply website/blog/2024-01-23-custom-pyfunc/index.md:6]: ahhh gotcha!

File: website/blog/2024-01-23-custom-pyfunc/index.md (hunk 2 above)
Comment (BenWilson2) [reply website/blog/2024-01-23-custom-pyfunc/index.md:345]: Looking at it again, it doesn't do any explanation that the functional code example in the next section shows. I'm going to remove this example, as it's not really useful.

**Pull request merged**